.github/workflows/build-it.yaml

```diff
@@ -37,6 +48,16 @@ jobs:
           df -h /
           tar -xzf ../maven-repo.tgz -C ~
           df -h /
+      - name: "Validate"
+        run: |
+          cd ${{ env.PROJECTS }}/cos-fleet-catalog-camel
+          
+          ./mvnw ${MAVEN_ARGS} clean install \
+            -Dcos.connector.container.tag=${{ inputs.tag }} \
+            -Pcontainer-build \
```

Comment:
would a profile with a "jar" package type be enough and quicker?

Reply:
it could be however we would need to create container images anyway in the subsequent step so I moved it here to avoid packaging the connector twice, maybe even better would be to explore if we can move the check in the validation phase